### DISCUSSION TITLE
Custom redirect URL when signin?

### DISCUSSION THREAD

**Discussion by socketopp** [nextauthjs/next-auth#10243] [Help]
Is it possible to customize the url that signin method returns? 

If my provider throw an exception I get the following url
`http://localhost:5173/auth/authorize/error?error=Configuration`

It seems that next-auth put '/auth' prefix.

**Comment by ndom91** [comment]
If yuo just want to change the `/auth` base path, then the `basePath` config option should do the trick . See: https://authjs.dev/reference/sveltekit#basepath
